ArcadiaParties.CQRS/Handlers/GetUsersOfDepartmentHandler.cs

```diff
@@ -0,0 +1,30 @@
+﻿using ArcadiaParties.CQRS.Queries;
+using ArcadiaParties.Data.Abstractions.DTOs;
+using ArcadiaParties.Data.Abstractions.Repositories;
+using AutoMapper;
+using MediatR;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ArcadiaParties.CQRS.Handlers
+{
+    internal class GetUsersOfDepartmentHandler : IRequestHandler<GetUsersOfDepartmentQuery, IEnumerable<UserForCalendarDTO>>
```

Comment:
It should be named another way, because it returns not users of department.

Reply:
Renamed